applications/luci-app-openwisp/root/etc/uci-defaults/40_luci-openwisp

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+uci -q batch <<-EOF >/dev/null
+	delete ucitrack.@openwisp[-1]
+	add ucitrack openwisp
+	set ucitrack.@openwisp[-1].init=openwisp_config
+	commit ucitrack
+EOF
+
+exit 0
```

Comment:
I'm not quite sure now, but I don't think we need this file any more, as we track the configuration file for changes with the following [line](https://github.com/openwisp/openwisp-config/blob/master/openwisp-config/files/openwisp.init#L74). In the init Script